changelog/unreleased/issue-2565

```diff
@@ -1,8 +1,9 @@
-Bugfix: Restic forget --keep-* options now interpret "-1" as "forever"
+Bugfix: Restic forget `--keep-*` options now support the value "unlimited"
```

Comment:
I suggest
~~~
Support "unlimited" in `forget --keep-*` options
~~~

Reply:
Is changed